### PR TITLE
feat(job table improvements): apply multiple improvements to jobs UI

### PR DESCRIPTION
**Pull request by jatinriverlane**
## Changes
* added support for dark mode in job details modal
* update styles in status chips to use theme constants instead of hardcoded values
* apply ellipsis in experiment name in chip in job details page

### CODE REVIEW

**Review by samiralavi — APPROVED** [review]
Thanks @jatinriverlane it looks good to me. Please check with @safeamiiir about showing status and result_code as they have different meanings.

File: aqueductcore/frontend/src/components/molecules/JobListTableCells/JobExperimentName/index.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ const ExperimentName = styled(Typography)`
     border: 1px solid #ccc;
     line-height: 1.375rem;
     background-color: #FFF;
+    color: ${({ theme }) => theme.palette.mode === "dark" ? theme.palette.grey[800] : theme.palette.grey[800]};
```
Comment (safeamiiir) [inline aqueductcore/frontend/src/components/molecules/JobListTableCells/JobExperimentName/index.tsx:13]: Is the condition not redundant here?

File: aqueductcore/frontend/src/components/molecules/JobListTableCells/JobExperimentName/index.tsx [hunk 2]
```diff
@@ -20,11 +21,21 @@ const ExperimentEid = styled(Typography)`
...
+    white-space: nowrap;
+`;
+
 function JobExperimentName({ name, eid }: { name: ExperimentData['title'], eid: ExperimentData['eid'] }) {
     return (
         <ExperimentNameChip container>
             <Grid item>
-                <ExperimentName noWrap>{name}</ExperimentName>
+                <ExperimentName noWrap sx={{
+                }}>
```
Comment (safeamiiir) [inline aqueductcore/frontend/src/components/molecules/JobListTableCells/JobExperimentName/index.tsx:36]:
```suggestion
                <ExperimentName noWrap>
```

**Review by safeamiiir — COMMENTED** [review]
Thanks Jatin!

File: aqueductcore/frontend/src/components/molecules/JobListTableCells/JobExperimentName/index.tsx (hunk 2 above)
Comment (jatinriverlane) [reply aqueductcore/frontend/src/components/molecules/JobListTableCells/JobExperimentName/index.tsx:36]: Cleaning this as well

**Review by safeamiiir — APPROVED** [review]

**Pull request merged**